source/__tests__/locApp.test.js

```diff
@@ -2,10 +2,10 @@ describe('Sinopia Profile Editor Homepage', () => {
 
   beforeAll(async () => {
     await page.goto('http://127.0.0.1:8000')
-  }, 100000)
+  }, 10000)
 
   it('displays "Profiles on Server" text on page', async () => {
     await expect(page).toMatch('Profiles on Server')
-  }, 100000)
+  }, 10000)
```

Comment:
yeah - same question about these timeouts - do we still need?

Reply:
The default timeout is 500, so I agree to remove these for now. If we run into problems we can add them back where needed. Also I agree with squashing the commits.